docker/single/Dockerfile

```diff
@@ -4,6 +4,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 WORKDIR /opt/arroyo/src
 ARG MOLD_ARCH
 ARG PROTO_ARCH
+ARG PROM_ARCH
```

Comment:
Might be worth finding some way to have one architecture arg that we convert to the various names.

Reply:
Agreed, should be possible with a small bash script?